### ISSUE TITLE
SVGs imported into PowerPoint are missing icons included via URLs

### ISSUE THREAD

**Issue by MLNW** [terrastruct/d2#1849]
When I drop the SVG of a diagram including icons into PowerPoint the icons themselves are missing. Here is an example diagram:

```d2
classes: {
  container_image: {
    shape: image
    icon: https://icon.icepanel.io/Technology/svg/Podman.svg
  }
  users: {
    shape: image
    icon: https://icons.terrastruct.com/essentials%2F359-users.svg
  }
}

image: Container Image {
  class: container_image
}

users: Users {
  class: users
}

image <- users
```

| Expected | Actual |
|-|-|
| ![image](https://github.com/terrastruct/d2/assets/20930814/14b19478-57a2-4a4c-8d9a-332903993d4a) | ![image](https://github.com/terrastruct/d2/assets/20930814/b5262792-a463-4870-aafa-26867f3e61b4) |
| VS code, Firefox ... | PowerPoint |

```shell
d2 --version
v0.6.3
```

**Comment by zachcp** [comment]
See also #1853

**Comment by alixander** [comment]
@MLNW I can't reproduce. I exported to ppt and this is result:

<img width="1584" alt="Screenshot 2024-04-17 at 7 44 06 PM" src="https://github.com/terrastruct/d2/assets/3120367/da4c756e-3eb9-43bf-b4a3-f6e5a14e8f4d">

Am I misunderstanding the issue? Do you mean you made an SVG with D2 and then imported that SVG into a ppt?

**Comment by MLNW** [comment]
Yes, I used the VS Code Extension to export the diagram as SVG and then imported that into PowerPoint. How would I use the --bundle parameter in VS Code?

**Comment by alixander** [comment]
Hm VSCode extension needs to support the option. Can you file this issue in https://github.com/terrastruct/d2-vscode/issues ?

**Comment by MLNW** [comment]
I just tried to use the work around via `--bundle` that you proposed. It leads to the same result. I also saw that the CLI already defaults to use bundle.

```
❯ d2
d2 v0.6.5
Usage:
...

Flags:
...
  -b, --bundle                 $D2_BUNDLE            when outputting SVG, bundle all assets and layers into the output file (default true)
...
```
